src/rgw/rgw_sal_motr.cc

```diff
@@ -3328,14 +3329,15 @@ int MotrObject::overwrite_null_obj(const DoutPrefixProvider *dpp)
   std::string obj_type = "simple object";
   rgw_bucket_dir_entry old_ent;
   bufferlist old_check_bl;
-  rc = this->fetch_null_obj(dpp, old_check_bl);
+  bool raise_error = false;
```

Comment:
1. During 1st PUT there will be no null reference key present in the bucket(obj^null), and if we try to read it, it returns -2 ret code.  In this case, this is expected we don't need to throw an error here.
2. But in get-object/delete-object API when we try to read null reference and if it is not present it should throw an error. 
So to handle this scenario in PUT operation(overwrite_null_obj() function ) we passing raise_error= false to fetch_null_obj() function, and by default raise_error=true, So in other API cases(delete, get ) it will throw an error.